src/pages/questionPage.js

```diff
@@ -6,8 +6,12 @@ import {
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
+// import { appendLinks } from '../utils/appendLinksUtil.js';
```

Comment:
Is this function not needed?

Reply:
I was getting a notification :
'appendLinks' is declared but its value is never read.'
I believe whoever added it didn't use it at the end! So I commented it out. to double check